apps/mn-synt-arbeidsforhold-service/src/main/java/no/nav/registre/testnorge/mn/syntarbeidsforholdservice/provider/IdentController.java

```diff
@@ -37,7 +38,7 @@
                     .badRequest()
                     .body("Kan ikke opprette for ident " + ident + "fordi personen allerede finnes AAREG (" + miljo + ").");
```

Comment:
`.body("Kan ikke opprette arbeidsforhold for ident " + ident + "fordi personen allerede finnes i AAREG (" + miljo + ").");`
Ikke kode du har endret på i PR, men kanskje like greit å endre i samme sleng? 🤓

Reply:
Veldig greit ja, skal oppdatere ;)